src/file_controller/mod.rs

```diff
@@ -285,11 +335,16 @@ impl Cache {
 
                 (text, pre, post)
             }
+            Ok(_) => unimplemented!(),
```

Comment:
Could this get a comment with a `FIXME` about what should be implemented here please?